app/src/main/java/app/accrescent/client/ui/SettingsScreen.kt

```diff
@@ -65,11 +78,23 @@ fun SettingsScreen(modifier: Modifier = Modifier, viewModel: SettingsViewModel =
                 onCheckedChange = { coroutineScope.launch { viewModel.setDynamicColor(it) } },
             )
         }
+        val networkTypeNames = listOf(R.string.all, R.string.unmetered, R.string.metered, R.string.not_roaming)
+        val networkTypeValues = listOf(NetworkType.CONNECTED, NetworkType.UNMETERED, NetworkType.METERED, NetworkType.NOT_ROAMING)
```

Comment:
I ask this out of ignorance, but is there any reason someone would want to restrict updates to only metered networks?

Reply:
I can't come up with a viable use case for that, however I don't see a reason to not add the possibility since there are no disadvantages from making it possible.

Reply:
I'd rather not have the metered option if there isn't a good reason for it. It can always be added later if a reason is found, but for now I think it would just be confusing.